integration-tests/gradle/pom.xml

```diff
@@ -192,118 +201,41 @@
         </dependency>
         <!-- END update-dependencies.sh -->
     </dependencies>
+    <build>
+        <testResources>
+            <testResource>
+                <directory>src/test/resources</directory>
+                <filtering>true</filtering>
+            </testResource>
+        </testResources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <configuration>
+                    <escapeString>\</escapeString>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
I don't think this is required? Because AFAICS this module has the default packaging (jar) which receives default plugin executions like surefire OOTB.

Reply:
yes, I agree, I just saw it in some other projects. I will remove it